Code/MD_class.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from inspect import getsource
+
+
+class MD_Simulation_1D_model_system:
+
+    def __init__(self, total_steps, potential, derivative_potential, time_step, collision_rate, mass, temperature, kboltz=0.008314, initial_position=0, initial_velocity=0):
+        self.potential = potential
+        self.derivative_potential = derivative_potential
+        self.time_step = time_step
+        self.collision_rate = collision_rate
+        self.mass = mass
+        self.temperature = temperature
+        self.total_steps = total_steps
+        self.kboltz = kboltz
+        self.initial_position = initial_position
+        self.initial_velocity = initial_velocity
+        self.total_time = self.time_step * self.total_steps
+
```

Comment:
units would be useful for the logfile

Reply:
mdtraj uses some unit class, maybe we can look into that ?

Reply:
Very good idea, thank you. I made a note and postponed that to later.